extensions/wikia/PhalanxII/classes/Phalanx.class.php

```diff
@@ -288,7 +317,10 @@ private function log( $action ) {
 		$log->addEntry(
 			$action,
 			$title,
-			wfMsgExt( 'phalanx-rule-log-details', array( 'parsemag', 'content' ), $this->data['text'], $types, $this->data['reason'] )
+			wfMsgExt(
+				'phalanx-rule-log-details', [ 'parsemag', 'content' ],
+				$this->data['text'], $types, $this->data['reason']
+			)
```

Comment:
I would have updated this to `wfMessage` but there aren't any clear docs on online to address `content` as an option.